app/components/user_item/user_item.tsx

```diff
@@ -146,15 +154,15 @@ const UserItem = ({
                         testID={`${userItemTestId}.current_user_indicator`}
                     />
                 }
-                {Boolean(user) &&
-                <Text
-                    style={style.rowUsername}
-                    numberOfLines={1}
-                    testID={`${userItemTestId}.username`}
-                >
-                    {` @${user!.username}`}
-                </Text>
-                }
+                {Boolean(user) && (
+                    <Text
+                        style={{...style.rowUsername, flexShrink: rowUsernameFlexShrink}}
```

Comment:
```suggestion
                        style={[style.rowUsername, {flexShrink: rowUsernameFlexShrink}]}
```

And maybe is best to memoize it

Reply:
Updated